src/DesignKit.tsx

```diff
@@ -2987,6 +2988,234 @@ const MessageSection: React.FC = () => {
+          <code>{'--elevation-bg-contrast-dark'}</code>
+          <code>{'--elevation-bg-backdrop'}</code>
+        </Columns>
+        <Columns>
+          <code>{'--elevation-bg-selected-light'}</code>
+          <code>{'--elevation-bg-selected-dark'}</code>
+          <code>{'--elevation-bg-hover-light'}</code>
+          <code>{'--elevation-bg-hover-dark'}</code>
+          <code>{'--elevation-bg-focused'}</code>
+          <code>{'--elevation-bg-border-light'}</code>
+          <code>{'--elevation-bg-border-dark'}</code>
+        </Columns>
+        <br />
+        <Paragraph>
+          Text elements and icons (<code>{'--text-secondary'}</code>).
+        </Paragraph>
+        <Paragraph>
+          The (<code>{'--text-contrast'}</code>) should be used for the tooltips.
+        </Paragraph>
+        <Columns>
+          <code>{'--text-title-light'}</code>
+          <code>{'--text-title-dark'}</code>
+          <code>{'--text-base-light'}</code>
+          <code>{'--text-base-dark'}</code>
+          <code>{'--text-secondary-light'}</code>
+        </Columns>
+        <Columns>
+          <code>{'--text-secondary-dark'}</code>
+          <code>{'--text-disabled-dark'}</code>
+          <code>{'--text-disabled-light'}</code>
+          <code>{'--text-link'}</code>
+          <code>{'--text-contrast-dark'}</code>
+          <code>{'--text-contrast-light'}</code>
+        </Columns>
+        <br />
+        <Paragraph>
+          The other way to use the Elevation module is to compose the CSS classes:
+        </Paragraph>
+        <Columns>
+          <code>{'%elevation-base-dark'}</code>
+          <code>{'%elevation-base-light'}</code>
+          <code>{'%elevation-strong-dark'}</code>
+          <code>{'%elevation-strong-light'}</code>
+          <code>{'%elevation-contrast-light'}</code>
+          <code>{'%elevation-contrast-dark'}</code>
+          <code>{'%elevation-border-light'}</code>
+        </Columns>
+        <Columns>
+          <code>{'%elevation-border-dark'}</code>
+          <code>{'%elevation-hover-light'}</code>
+          <code>{'%elevation-hover-dark'}</code>
+          <code>{'%elevation-selected-light'}</code>
+          <code>{'%elevation-selected-dark'}</code>
+          <code>{'%elevation-backdrop'}</code>
+        </Columns>
+        <Columns>
+          <code>{'%text-title-light'}</code>
+          <code>{'%text-title-dark'}</code>
+          <code>{'%text-secondary-light'}</code>
+          <code>{'%text-secondary-dark'}</code>
+          <code>{'%text-disabled-light'}</code>
+          <code>{'%text-disabled-dark'}</code>
+          <code>{'%text-link-light'}</code>
+        </Columns>
+        <Columns>
+          <code>{'%elevation-shadow-small'}</code>
+          <code>{'%elevation-shadow-medium'}</code>
+          <code>{'%elevation-shadow-large'}</code>
+        </Columns>
+        <br />
+        <Paragraph>
+          base elevation without hover interaction (for elements elements at level 1)
```

Comment:
nit: These example titles should be capitalized and bolded.
fix: duplicate "elements"
q: it seems like this elevation system doesn't really have a concept of levels anymore, so i'm not sure what "level 1" means here